src/pages/RepoPage/SettingsTab/tabs/BadgesAndGraphsTab/Badges/Badges.tsx

```diff
@@ -0,0 +1,150 @@
+import PropTypes from 'prop-types'
+import { useMemo, useState } from 'react'
+import { useParams } from 'react-router-dom'
+
+import config from 'config'
+
+import { Branch, useBranches } from 'services/branches'
+import A from 'ui/A'
+import Select from 'ui/Select'
+import SettingsDescriptor from 'ui/SettingsDescriptor'
+import TokenWrapper from 'ui/TokenWrapper'
+
+type UseBadgesProps = {
+  graphToken: string
+  provider: string
+  owner: string
+  repo: string
+  branch: Branch
+}
+
+const useBadges = ({
```

Comment:
`m`: This function isn't using any other hooks within it, so it itself shouldn't be a hook (i.e. using the word `use`), and just a normal function like `createBadges`.

Reply:
🧠 another thing I didn't know TY

Reply:
There is actually an `eslint` rule for it somewhere. I think at the moment it's only if you use a React hook inside of a function that doesn't have the name `use` at the beginning.